### PR TITLE
added 20.9.2 rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.9.2.md [hunk 1]
```diff
@@ -0,0 +1,1163 @@
...
+##### Bonusly - AutoGratitude  
+AutoGratitude is a playbook to give back a positive gratitude to security engineers and developers when they successfully complete an SLA
+#### Scripts
+##### IncOwnerToBonuslyUser  
+This script gets the email address of the incident owner and then returns the incident owner username in Bonusly.
+
+
+---
+
+### New: CaseManagement-Generic Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.2.md:81]: Add the following pack description one line below:
Beta Pack built by the Cortex Customer Success Team to provide quick and valuable deployment of XSOAR for the main use of Case Management.

File: content-repo/extra-docs/releases/20.9.2.md [hunk 2]
```diff
@@ -0,0 +1,1160 @@
...
+---
+
+### New: Bmc Helix Remedyforce Pack v1.0.0
+#### Classifiers
+##### BMCHelixRemedyforce - Classifier  
+Classifies BMC Helix Remedy force incidents.
+
+##### BMCHelixRemedyforce - Incoming Mapper  
+Maps incoming BMC Helix Remedy force incident fields.
+#### IncidentFields
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:19]:
```suggestion
#### Incident Fields
```

File: content-repo/extra-docs/releases/20.9.2.md [hunk 3]
```diff
@@ -0,0 +1,1160 @@
...
+- **Bmc Remedyforce Resolution**
+- **Bmc Remedyforce Responded Date**
+- **Bmc Remedyforce Service**
+- **Bmc Remedyforce Service Offering**
+- **Bmc Remedyforce Service Request**
+- **Bmc Remedyforce Staff**
+- **Bmc Remedyforce Status**
+- **Bmc Remedyforce Template**
+- **Bmc Remedyforce Urgency**
+#### IncidentTypes
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:48]:
```suggestion
#### Incident Types
```

File: content-repo/extra-docs/releases/20.9.2.md [hunk 4]
```diff
@@ -0,0 +1,1160 @@
...
+Beta pack built by the Cortex Customer Success Team to provide quick and valuable deployment of XSOAR for case management.
+
+#### Dashboards
+##### My Incidents  
+
+##### Case Management Implementation Guide  
+
+##### Incidents Overview  
+
+#### IncidentTypes
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:91]:
```suggestion
#### Incident Types
```

File: content-repo/extra-docs/releases/20.9.2.md [hunk 5]
```diff
@@ -0,0 +1,1160 @@
...
+##### Gophish  
+Gophish is a powerful, open-source phishing framework that makes it easy to test your organization's exposure to phishing. 
+
+
+---
+
+### New: TrendMicro Cloud App Security Pack v1.0.0
+##### TrendMicroCAS - incoming - mapper  
+Maps incoming Trend Micro Cloud App Security (CAS) fields.
+#### IncidentTypes
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:153]:
```suggestion
#### Incident Types
```

File: content-repo/extra-docs/releases/20.9.2.md [hunk 6]
```diff
@@ -0,0 +1,1160 @@
...
+---
+
+### Anomali ThreatStream Pack v1.0.2
+#### Integrations
+##### Anomali ThreatStream v2  
+Rewrote the ***threatstream-import-indicator-without-approval*** command and removed the deprecated flag.
+
+---
+
+### ApiModules Pack v1.1.4
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:203]: remove all api modules from the release notes

File: content-repo/extra-docs/releases/20.9.2.md [hunk 7]
```diff
@@ -0,0 +1,1160 @@
...
+---
+
+### DeepInstinct Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### Deep Instinct  
+Added a description to the integration.
+
+---
+
+### Deprecated Content Pack v1.6.6
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:534]: remove rn of all deprecated content pack

File: content-repo/extra-docs/releases/20.9.2.md [hunk 8]
```diff
@@ -0,0 +1,1160 @@
...
+##### Microsoft Management Activity API (O365 Azure Events)  
+- Maintenance and stability enhancements.
+- Updated the Docker image to the latest version.
+- Added the *timeout* argument to the ***ms-management-activity-list-content*** to fetch larger amounts of data.
+- Fixed a bug that occurred when resubscribing to a subscription. 
+- Updated the Docker image to *demisto/pyjwt3:1.0.0.8871*.
+
+---
+
+### MicrosoftCloudAppSecurity Pack v1.0.3
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.2.md:853]:
```suggestion
### Microsoft Cloud App Security Pack v1.0.3
```

**Review by Itay4 — APPROVED** [review]

**Pull request merged**